descriptors/ModuleDescriptor-template.json

```diff
@@ -307,15 +307,6 @@
             "inventory-storage.authority-source-files.item.post"
           ]
         },
-        {
-          "methods": [
-            "PUT"
-          ],
-          "pathPattern": "/authority-source-files/{id}",
-          "permissionsRequired": [
-            "inventory-storage.authority-source-files.item.put"
-          ]
-        },
```

Comment:
Same for this interface version should be bumped. Tickets for dependant modules should be created. PR could be merged only when related dependencies updated.